llvm/unittests/CodeGen/GlobalISel/CSETest.cpp

```diff
@@ -75,6 +75,138 @@ TEST_F(AArch64GISelMITest, TestCSE) {
   auto MIBUnmerge2 = CSEB.buildUnmerge({s32, s32}, Copies[0]);
   EXPECT_TRUE(&*MIBUnmerge == &*MIBUnmerge2);
 
+  // Check G_FADD
```

Comment:
In a cleanup you could use buildInstr and loop over the opcodes, these are all going to behave the same way